src/utils/streams.js

```diff
@@ -124,20 +137,10 @@ export class Concatenator extends Writable {
     callback(err);
   }
 
-  _final(callback: (error?: Error) => mixed) {
-    this.log.trace('_final()');
-    this.contents = Buffer.concat(this.chunks);
-    this.chunks.length = 0;
-    callback();
-  }
-
   transferContents(): Buffer {
```

Comment:
I kept this one for tests, as it's still useful in that context. But we shouldn't use it for the production code.

I wonder if I should add a check for `NODE_ENV==='test'`... That wouldn't work when running tests obviously but we'd see it when running integration tests. Thoughts?